dev/scripts/testTsVersions.ts

```diff
@@ -1,15 +1,14 @@
-import { rmSync, writeFileSync } from "node:fs"
-import { fromPackageRoot, readPackageJson } from "../runtime/fs.js"
-import { shell } from "../runtime/shell.js"
+import { readFileSync, rmSync, writeFileSync } from "node:fs"
+import { fromHere, fromPackageRoot } from "../attest/src/runtime/fs.js"
+import { shell } from "../attest/src/runtime/shell.js"
 
 const versions: { [k: string]: string } = {
     // Disabled because of the changes made to accommodate StackBlitz. Reenable:
     // https://github.com/arktypeio/arktype/issues/659
     // "4.8": "16",
     "4.9": "17"
 }
-
-const rootJson = readPackageJson(fromPackageRoot())
+const rootJson = JSON.parse(readFileSync(fromHere("package.json"), "utf-8"))
```

Comment:
We can't use readJson in cases like this?

I know you mentioned something about that but I don't remember why it would be a problem.